src/main/java/net/pms/formats/audio/MPGAudio.java

```diff
@@ -33,6 +33,10 @@ public Identifier getIdentifier() {
 	 */
 	@Override
 	public String[] getSupportedExtensions() {
-		return new String[] { "mpa", "mp2" };
+		return new String[] {
+			"mpa",
+			"mp1",
```

Comment:
@Nadahar Houra, i discovered an easy way to see what mean all these strange Emoji :smile: